app/views/galleries/index.rss.builder

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+self.class.send(:include, GalleryDisplayingView)
+
+def entry_summary(gallery)
+  image_count = gallery.images.count
+  unless image_count.zero?
+    presenter = presenter_for_gallery_image(gallery.images.first)
+    image_url = presenter.field_value('edmPreview')
+  end
+  image_url ||= @hero_image && @hero_image.file.present? ? @hero_image.file.url : nil
+
+  %[
+    <h2>#{gallery.title} <span>(#{image_count} #{t('global.labels.images')})</span></h2>
+    <img src=\"#{image_url}\"/>
```

Comment:
Could you move this into either an `<enclosure>` or `<media:thumbnail>` on each item?.